src/main/java/com/st/lms/models/Author.java

```diff
@@ -0,0 +1,51 @@
+package com.st.lms.models;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+@JsonIgnoreProperties({"hibernateLazyInitializer", "handler"})
+@Entity
+@Table(name = "tbl_author")
+public class Author {
+	
+	@Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+	private int authorId;
+	
+	@Column(name="authorName")

```

Comment:
You don't need to specify the name of the column if it's the same as the name of the field (unless Hibernate translates the field name into a different format first, in which case document that).